fighthealthinsurance/fax_polling_actor.py

```
@@ -1,54 +1,72 @@
 import ray
 from fighthealthinsurance.fax_actor import FaxActor
-import time
 import asyncio
+import logging
+from fighthealthinsurance.fax_actor import FaxActor
```

Ruff (0.8.2)

Redefinition of unused `FaxActor` from line 2

Remove definition: `FaxActor`

(F811)